Libraries/LibWeb/HTML/Numbers.cpp

```diff
@@ -56,13 +56,17 @@ Optional<StringView> parse_integer_digits(StringView string)
 }
 
 // https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#rules-for-parsing-integers
-Optional<i32> parse_integer(StringView string)
+Optional<::Crypto::SignedBigInteger> parse_integer(StringView string)
```

Comment:
Is there a case you've found where a caller of this function would expect a value larger than 2147483647 to be valid? Personally, I'd rather not use BigInts in these functions if it's not strictly necessary.

Reply:
See this discord conversation https://discord.com/channels/1247070541085671459/1247090064480014443/1319377687567011942